### ISSUE TITLE
Unexpected token import

### ISSUE THREAD

**Issue by ArchOrn** [ripjar/material-datetime-picker#80]
Hi,

When I try to use your library with SystemJS by importing it like so:

```
import MaterialDatePicker from 'material-datetime-picker';
```

or

```
import MaterialDateTimePicker from 'material-datetime-picker';
```

I got this error:

```
Error: (SystemJS) SyntaxError: Unexpected token import
```

Do you have any idea about where this could come from? Because after looking at the code, I think the README is pointing to the wrong component.

Thanks for your time,

Raphaël
# Environment
## Browser
- Chrome 51 (Windows, Mac)
## Library
- v1.0.1 (via jspm => npm)

**Comment by Jivings** [comment]
You'll probably need the systemjs [babel plugin][1]. The library is written in ES6, so if you don't have a transpiler then things wont work correctly!

[1]: https://github.com/systemjs/plugin-babel

**Comment by ArchOrn** [comment]
My code is also written in ES6 and I already use Babel as a transpiler. The core of my project is using mithriljs. I will try to reproduce this error in a fresh environment with JsFiddle for you to see.

Thank you for your time.

**Comment by Jivings** [comment]
Yes, that will be useful, thanks! i don't know why your transpiler is moaning about `import` statements. Are you using them elsewhere in your project? I think you need the `[es2015]` babel preset for imports, perhaps that is what is missing. 

I will add it to the project as well.

**Comment by simonq-nvc** [comment]
I am also experiencing this issue and I have the `[es2015]` babel preset. Is there any solutions to this?

**Comment by Jivings** [comment]
@simonq-nvc Is the rest of your project ES6? Do other import statements complain? What version of babel are you using?

**Comment by orwldn** [comment]
same here. other imports do work though. 

babel packages:
```
    "babel-preset-stage-2": "^6.11.0",
    "babel-plugin-transform-object-rest-spread": "^6.8.0",
    "babel-core": "^6.7.4",
    "babel-loader": "^6.2.4",
    "babel-preset-es2015": "^6.6.0",
    "babel-preset-react": "^6.5.0",
```

**Comment by Jivings** [comment]
@orwldn can you share your `.babelrc`? I think by default babel will not compile anything in `node_modules`.

**Comment by orwldn** [comment]
it only includes `transform-object-rest-spread`

**Comment by RyanThomasMusser** [comment]
+1

**Comment by Jivings** [comment]
I have tried to fix this but it seems that browserify/babel makes it really confusing the build standalone modules. I will continue trying today.

**Comment by brickyang** [comment]
@Jivings That's why I don't think use `import` in a npm module is a good idea

**Comment by Jivings** [comment]
@brickyang Agreed. There's should be a compiled dist version available.

**Comment by aj1310** [comment]
Inside the create-react-app environment, it falls into the same error. Any recommendations?

**Comment by tdfairbrother** [comment]
+1

**Comment by ncasich** [comment]
I suppose if there were used require ensure instead of imports at least webpack could build it.

Is there is currently any way to launch it?

**Comment by jwhitfieldseed** [comment]
The version on `master` (since merging #99) can be imported by Webpack (including create-react-app) and SystemJS/JSPM. 

I'll cut a `2.0.0` release in the next few days. Thanks for your patience!

**Comment by jwhitfieldseed** [comment]
Can you try `2.0.0-pre`? 

The build issues should be resolved there.

**Comment by jwhitfieldseed** [comment]
Fixed in `2.0.0`.
